CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
-## 1.10.0
+## 1.9.2
 Release built: _not released yet_
 
+### What’s new?
+- Added support for pre-allocated, non-persisted accounts in the `/state/account/page/resource-preferences` and `/state/account/page/authorized-depositors` endpoints.
+
+### Bug fixes
+- Fixed a typo in the value `StoryOnlyForUserTransactionsAndEpochChanges` (replacing `Story` with `Store`) for the configuration entries `DataAggregator__Storage__StoreTransactionReceiptEvents` and `DataAggregator__Storage__StoreReceiptStateUpdates`. It now supports both values:
+  - `StoreOnlyForUserTransactionsAndEpochChanges`
+  - `StoryOnlyForUserTransactionsAndEpochChanges`
+- Fixed `/state/entity/details` when querying with `native_resource_details: true` option enabled for 
```

Comment:
This should probably be top of the list as it's most important. Perhaps we should also be explicit that this is the 500 people were seeing, e.g.:

```suggestion
- Fixes a HTTP 500 response issue when querying the `/state/entity/details` endpoint with the `native_resource_details: true` opt-in for:
```